coderd/searchquery/search_test.go

```diff
@@ -243,7 +270,11 @@ func TestSearchWorkspace(t *testing.T) {
 		c := c
 		t.Run(c.Name, func(t *testing.T) {
 			t.Parallel()
-			values, errs := searchquery.Workspaces(c.Query, codersdk.Pagination{}, 0)
+			db := dbmem.New()
+			if c.Setup != nil {
+				c.Setup(t, db)
+			}
+			values, errs := searchquery.Workspaces(context.Background(), db, c.Query, codersdk.Pagination{}, 0)
```

Comment:
Can we not use the traditional way of getting a DB in tests so this can run against postgres too?

Reply:
I'd prefer not to for this. What would be even better is if I used the mock database, but I haven't messed with that at all yet.